src/lib/queue.js

```diff
@@ -187,8 +187,17 @@ export function setRedisConnectionName(queue, connectionName) {
 }
 
 export default function newQueue(queName) {
-  const queue = new Queue(queName, `redis://${host}:${port}`, redisOpts);
+  const queue = new Queue(queName, `redis://${host}:${port}`, {
+    ...redisOpts,
+    maxRetriesPerRequest: 50, // Adjust this value as needed
```

Comment:
Should this be 50? Seems large.

Reply:
we can reduce it, what would you this would be a good value?

Reply:
I might be way off on this but like 3 or 5. My worry is some large job will get stuck and keep retrying 50 times.

Reply:
```suggestion
    maxRetriesPerRequest: 15, // Adjust this value as needed
```